### ISSUE TITLE
KeyNotFoundException: The given key 'Ngid' was not present in the dictionary.

### ISSUE THREAD

**Issue by szmcdull** [flamencist/Linux.ProcessManager#1]
```
var processes = ProcessManager.GetProcessInfos();

KeyNotFoundException: The given key 'Ngid' was not present in the dictionary.
   at System.ThrowHelper.ThrowKeyNotFoundException[T](T key)
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Linux.ProcFs.TryParseStatusFile(String statusFilePath, ParsedStatus& result, ReusableTextReader reusableReader)
   at Linux.ProcessManagerImpl.CreateProcessInfo(Int32 pid, ReusableTextReader reusableReader)
   at System.Linq.Enumerable.SelectArrayIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToArray()
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
```

The environment is Debian GNU/Linux over WSL (Windows Subsystem Linux).

**Comment by flamencist** [comment]
Could provide additional info: windows build version, debian version

**Comment by szmcdull** [comment]
Windows 10 1809 (Microsoft Windows [Version 10.0.17763.503])
Debian GNU/Linux 9 \n \l
cat debian_version: 9.9
Linux DESKTOP-NUIQLJ5 4.4.0-17763-Microsoft #379-Microsoft Wed Mar 06 19:16:00 PST 2019 x86_64 GNU/Linux

**Comment by szmcdull** [comment]
$ cat status
```
Name:   bash
State:  S (sleeping)
Tgid:   67
Pid:    67
PPid:   66
TracerPid:      0
Uid:    1000    1000    1000    1000
Gid:    1000    1000    1000    1000
FDSize: 4
Groups:
VmPeak: 0 kB
VmSize: 14608 kB
VmLck:  0 kB
VmHWM:  0 kB
VmRSS:  3272 kB
VmData: 0 kB
VmStk:  0 kB
VmExe:  1024 kB
VmLib:  0 kB
VmPTE:  0 kB
Threads:        1
SigQ:   0/0
SigPnd: 0000000000000000
ShdPnd: 0000000000000000
SigBlk: 0000000000000000
SigIgn: 0000000000000000
SigCgt: 0000000000000000
CapInh: 0000000000000000
CapPrm: 0000000000000000
CapEff: 0000000000000000
CapBnd: 0000001fffffffff
Cpus_allowed:   00000001
Cpus_allowed_list:      0
Mems_allowed:   1
Mems_allowed_list:      0
voluntary_ctxt_switches:        150
nonvoluntary_ctxt_switches:     545
```

**Comment by flamencist** [comment]
@szmcdull should be fixed greater or equal 1.5.3 version
